### PR TITLE
Make sure watchman is installed and open simulator

### PR DESCRIPTION
**Pull request by mfpiccolo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,9 +25,16 @@ run-ios: .npminstall config/config.secret.json
...
 	fi
+	@if ! [ $(shell command -v watchman) ]; then \
+		echo "watchman is not installed"; \
+		exit 1; \
+	fi
+
 	@echo Running iOS app in development
 
 	npm run run-ios
+	open -a Simulator
```
Comment (thomchop) [inline Makefile]: Won't `npm run run-ios` already open the simulator? Is the purpose of this added line to trigger an obvious failure if the simulator isn't already installed? If so, let's fire up the simulator before we call `run-ios`.

Comment (mfpiccolo) [inline Makefile]: It doesn't open on several of the computers that I tried to with the make command.  But yeah it could be moved above.

**Comment by coreyhulen** [comment]
close and re-opening so the build will fire.

File: Makefile (hunk 1 above)
Comment (thomchop) [inline Makefile]: Don't we also want to check for `watchman` in `run-android`? I'll add that real quick.

**Comment by thomchop** [comment]
Hang on, it appears that this is no longer working: my simulator is not being opened. I'll investigate. Maybe we need to open it before we call run-ios.

**Comment by thomchop** [comment]
It looks like the problem was caused by me installing watchman without the xcode CLT installed (or at least not the tools that match my currently selected version of xcode).

**Pull request merged**